unicodetools/.classpath

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path=""/>
-	<classpathentry combineaccessrules="false" kind="src" path="/cldr-code"/>
+	<classpathentry combineaccessrules="false" exported="true" kind="src" path="/cldr-code"/>
```

Comment:
is this change on purpose?
what does it do?

Reply:
Hmm, Eclipse made this change. I can undo it and see if stuff still works

Reply:
Looks like removing it doesn't break anything. Unsure why eclipse added it,